frontend/src/lib/WorkflowParser.ts

```diff
@@ -361,6 +362,12 @@ export default class WorkflowParser {
         key: pathParts.slice(1).join('/'),
         source: StorageService.HTTPS,
       };
+    } else if (strPath.startsWith('/')) {
+      return {
+        bucket: StorageService.FILE,
```

Comment:
Is it possible parsing the volume name from the path and other metadata info in argo workflow?